processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/DecatonTaskRetryQueueingProcessor.java

```diff
@@ -71,6 +72,7 @@ public void process(ProcessingContext<byte[]> context, byte[] serializedTask)
             } else {
                 metrics.retryQueueingFailed.increment();
             }
+            metrics.retryTaskRetries.record(nextRetryCount);
```

Comment:
Isn't it better to record before sending produce request instead of inside callback?
Because we may want to know "retry is attempted over X times"  regardless queueing is completed or not

Reply:
Thank you, that sounds reasonable. Updated in https://github.com/line/decaton/pull/178/commits/58e93c5b33e5cbf0ce585b723aaaa018576a2dfe